server/controllers/admin.py

```diff
@@ -440,7 +467,7 @@ def templates(cid, aid):
         if files:
             templates = {}
             for template in files:
-                templates[template.filename] = str(template.read(), 'latin1')
+                templates[template.filename] = str(template.read(), 'utf-8')
```

Comment:
ok-client assumes files are utf-8 text. I think it's reasonable that web uploads and templates have the same restriction